util/chplenv/chpl_tasks.py

```diff
@@ -20,7 +21,7 @@ def get():
         # use the qthreads it provides even if the user has an older CCE loaded
         using_qthreads_incompatible_cce = False
         if compiler_val == 'cray-prgenv-cray':
-            if (utils.get_compiler_version(compiler_val) < CompVersion('8.4') and
+            if (get_compiler_version(compiler_val) < CompVersion('8.4') and
                     not using_chapel_module()):
```

Comment:
Need to import using_chapel_module from utils. huh, I guess that was broken before too